aiven/client/cli.py

```diff
@@ -3059,6 +3059,86 @@ def mirrormaker__replication_flow__delete(self):
             self.args.target_cluster,
         )
 
+    @arg.project
+    @arg.service_name
+    def service__flink__table__list(self):
+        """List created Flink tables"""
+        project_name = self.get_project()
+        self.print_response(self.client.list_flink_tables(project_name, self.args.service_name))
+
+    @arg.project
+    @arg.service_name
+    @arg("integration_id", help="Service integration ID")
+    @arg("-n", "--table_name", required=True, help="Table name")
+    @arg("-kt", "--kafka_topic", required=False, help="Topic name, used as a source/sink. (Kafka integration only)")
+    @arg("-dt", "--jdbc_table", required=False, help="Table name in Database, used as a source/sink. (PG integration only)")
```

Comment:
In addition to the hyphen comment, it's better to avoid multi-letter shorthand versions. In this case perhaps just omit the shorthand versions altogether.